serenata_toolbox/federal_senate/dataset.py

```diff
@@ -49,7 +53,9 @@ def translate(self):
             try:
                 self._translate_file(csv_path)
             except FileNotFoundError as file_not_found_error:
-                print("While translating, Seranata Toolbox didn't find file: {} \n{}".format(csv_path, file_not_found_error))
+                print("While translating, Seranata Toolbox didn't find file: {} \n{}".format(
+                    csv_path,
+                    file_not_found_error))
```

Comment:
It feels like you missed one space, and we have a matter os style: 

Either… 

```python
print("While translating, Seranata Toolbox didn't find file: {} \n{}".format(file_path,
                                                                             file_not_found_error))
```

…or…

```python
print("While translating, Seranata Toolbox didn't find file: {} \n{}".format(
    file_path,
    file_not_found_error)
)
```